src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -5,4 +5,16 @@
  */
 public class HelloWorld {
 
+    public static void main(String[] args) {
+
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i < figures.length; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+            figures[i].print();
+
+        }
+
+    }
+
 }
```

Comment:
unnecessary empty lines
```suggestion
    public static void main(String[] args) {
        FigureSupplier figureSupplier = new FigureSupplier();
        Figure[] figures = new Figure[6];
        for (int i = 0; i < figures.length; i++) {
            figures[i] = figureSupplier.getRandomFigure();
            figures[i].print();
        }
    }
}
```